content/en/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -254,4 +254,8 @@ If the above command corrects the error, you can automate the step by adding the
 If you add the `lifecycle` section show above to your Pod spec, the Pod will execute the commands listed to restart the `netlogon` service until the `nltest.exe /query` command exits without error.
 
 ## GMSA limitations
-When using the [ContainerD runtime for Windows](/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#cri-containerd) accessing restricted network shares via the GMSA domain identity fails. The container will recieve the identity of and calls from `nltest.exe /query` will work.  It is recommended to use the [Docker EE runtime](/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#docker-ee) if access to network shares is required.  The Windows Server team is working on resolving the issue in the Windows Kernel and will release a patch to resolve this issue in the future. Look for updates on the [Microsoft Windows Containers issue tracker](https://github.com/microsoft/Windows-Containers/issues/44).
+When using the [ContainerD runtime for Windows](/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#cri-containerd) accessing restricted network shares via the GMSA domain identity fails. The container will receive the identity of and calls from `nltest.exe /query` will work.  It is recommended to use the [Docker EE runtime](/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#docker-ee) if access to network shares is required.  
+
+This has been fixed, For Windows Server 2004, and this fix will land in the 11C Windows Patch, with pending information about patch release info for newer windows editions (i.e. 2019).
```

Comment:
We usually [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future), (mentioned [previously](https://github.com/kubernetes/website/pull/24884#discussion_r518266500)).

This still looks like a statement about the future. It might be OK to make an exception to the style guide - @jayunit100 can you explain why that exception is appropriate here?

Reply:
If you remove that statement about the future fix, I think this could be OK to merge (but I'd like tech review from SIG Windows).

Reply:
It would be great if we wrap these long lines to shorter ones. It is hard to check what has been changed.